src/libopal.c

```diff
@@ -2312,6 +2312,13 @@ gen_asm_code (node_s *ast)
       gen_asm_code(ast->left);
       add_asm_code(ast->node_type, 0, NULL);
       break;
+    case nd_Ident:
+    case nd_Integer:
+    case nd_String:
```

Comment:
Get `location_offset` of string by calling `add_str` and add asm code `asm_Push`